ada_feeding/ada_feeding/trees/move_to_tree.py

```diff
@@ -31,24 +31,16 @@ class MoveToTree(ActionServerBT, ABC):
 
     def __init__(
         self,
-        action_type_class_str: str,
     ) -> None:
-        """
-        Initializes tree-specific parameters.
```

Comment:
I'm not a fan of having no docstring on a function. If we ever automatically generate documentation, then a function without a docstring would have nothing.

Can you keep the docstring and just remove the decription of `action_type_class_str`?

Reply:
Done